src/index.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const state = {
+  temp: 75,
+};
+
+const changeTemp = () => {
+  //helper function
+  const bigTempContainer = document.querySelector('#big_temp');
+  bigTempContainer.textContent = `${state.temp}`;
+  const body = document.body;
+
+  changeTempColor(body);
+};
+
+const increaseTemp = () => {
+  state.temp += 1;
+  changeTemp();
+};
+
+const decreaseTemp = () => {
+  state.temp -= 1;
+  changeTemp();
+};
+
+const changeTempColor = (body) => {
+  const landscape = document.querySelector('#landscape');
+
+  if (state.temp > 80) {
+    body.className = 'hot';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.temp > 70) {
+    body.className = 'warm';
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.temp > 60) {
+    body.className = 'moderate';
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.temp > 50) {
+    body.className = 'cold';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else {
+    body.className = 'very-cold';
+    landscape.textContent = '🏔❄️☃️⛷🏔❄️☃️⛷🏔❄️☃️⛷';
+  }
+};
+
+const changeCity = (input) => {
+  const newCity = document.querySelector('#city').value;
+  const cityContainer = document.querySelector('#city-container');
+  cityContainer.textContent = `for ${newCity}`;
+};
+
+const getLatLon = () => {
+  const newCity = document.querySelector('#city').value;
+  axios
+    .get('http://Localhost:5000/location', {
```

Comment:
You could put the URL in a constant variable and reference it here, like: 
const LOCATION_API_URL = http://localhost:5000/location;